circup/class_definitions.py

```diff
@@ -0,0 +1,345 @@
+import os
+from urllib.parse import urljoin, urlparse
+
+import click
+import requests
+from semver import VersionInfo
+
+from circup.shared import DATA_DIR, PLATFORMS, REQUESTS_TIMEOUT, BAD_FILE_FORMAT, tags_data_load, \
+    get_latest_release_from_url
+from circup.backends import WebBackend
+from circup.logging import logger
+
+#: Flag to indicate if the command is being run in verbose mode.
+VERBOSE = False
+
+#: The version of CircuitPython found on the connected device.
+CPY_VERSION = ""
+
+
+class Bundle:
```

Comment:
I'd just split each of these classes into individual files of the same name. They'll be easier to find then.

Reply:
Bundle and Module are moved into their own respectively named files with the latest commits.